doc/source/examples/multiphysics/laser-heating/laser-heating.rst

```diff
@@ -0,0 +1,231 @@
+            set h	              = 5
+            set Tinf	      = 20
+            set emissivity        = 0.4
+        end
+            subsection bc 4
+        	set id = 4
+    	   set type	      = convection-radiation
+            set h	              = 5
+            set Tinf	      = 20
+            set emissivity        = 0.4
+        end
+            subsection bc 5
+        	set id = 5
+    	   set type	      = convection-radiation
+            set h	              = 5
+            set Tinf	      = 20
+            set emissivity        = 0.4
+        end
+    end
+
+
+The ``multiphysics`` subsection enables to turn on (``true``) 
+and off (``false``) the physics of interest. Here only ``heat transfer`` is enabled.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+	    set heat transfer          = true
+    end 
+    
+
+In the ``laser parameters`` section, the parameters of the laser model are defined. The exponential decaying model `[1] <https://doi.org/10.1016/j.matdes.2018.01.022>`_ is used to simulate the laser heat source. In the exponential decaying model, the laser heat flux is calculated using the following equation:
+
+    .. math:: 
+        q(x,y,z) = \frac{\eta \alpha P}{\pi r^2 \mu} \exp{(-\eta \frac{r^2}{R^2})} \exp{(- \frac{|z|}{\mu})}
+
+
+where :math:`\eta`, :math:`\alpha`, :math:`P`, :math:`R`, :math:`\mu`, :math:`r` and :math:`z` denote concentration factor, absorptivity, laser power, beam radius, penetration depth, radial distance from the laser focal point, and axial distance from the laser focal point, respectively. These parameters are explained in more detail in `laser parameters <https://lethe-cfd.github.io/lethe/parameters/cfd/laser_heat_source.html>`_.
+
+
+.. note:: 
+    The scanning path of the laser is defined using a Function expression in the ``path`` subsection. Here the laser ``path`` is a function of time, and changes its direction twice during laser operation.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Laser parameters
+    #---------------------------------------------------
+    subsection laser parameters
+        	set enable = true
+        	set concentration factor = 50
+        	set power = 3
+        	set absorptivity = 0.6
+        	set penetration depth = 0.00005
+        	set beam radius = 0.00005
+        	set start time = 0
+        	set end time = 0.003
+        	set beam orientation = z-
+        	subsection path
+        		set Function expression = if(t<0.001, 0.5 * t, if(t<0.002, 0.0005, if(t<0.003 , 0.0005-0.5 * (t-0.002), -1))); if(t<0.001, 0.00025, if(t < 0.002, 0.00025 - 0.5 * (t-0.001) , if(t < 0.003 , -0.00025, -1))) ; 0.0003
+        	end
+    end    
+
+
+In the ``mesh adaptation`` subsection, we choose a mesh refinement based on the variable ``temperature``.
```

Comment:
Shahab, I think it would be nice if you link this part with the "Mesh Adaptation Control" page.